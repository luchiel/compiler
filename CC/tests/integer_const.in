198
  256
  066
0xAAA
0XC0CAC01A
7888