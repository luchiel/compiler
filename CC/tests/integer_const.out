198
  256
  066
0xAAA
0XCOCACO1A
7888